src/blosc2/ndarray.py

```diff
@@ -1005,6 +1094,20 @@ def sum(ndarr: NDArray | NDField | blosc2.C2Array, axis=None, dtype=None, keepdi
     References
     ----------
     `np.sum <https://numpy.org/doc/stable/reference/generated/numpy.sum.html>`_
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> # Example array
+    >>> array = np.array([[1, 2, 3], [4, 5, 6]])
+    >>> # Sum all elements in the array (axis=None)
+    >>> total_sum = array.sum()
+    >>> print("Sum of all elements:", total_sum)
+    21
+    >>> # Sum along axis 0 (columns)
+    >>> sum_axis_0 = array.sum(axis=0)
+    >>> print("Sum along axis 0 (columns):", sum_axis_0)
+    [5 7 9]
```

Comment:
You are showing a NumPy example, not a Blosc2 one.  Also, please use the `blosc2.sum(array)` versions instead of methods (`array.sum()`).

Reply:
Done!